v19.2/cockroachcloud-connect-to-your-cluster.md

```diff
@@ -33,7 +33,7 @@ Use the Console to authorize networks:
 4. (Optional) Enter a descriptive name for the network.
 
 5. From the **Network** dropdown, select:
-   - **New Network** to authorize your application server’s network or application server’s network. Enter the public IPv4 address of the machine in the **Network** field.
+   - **New Network** to authorize your local machine's network or application server’s network. Enter the public IPv4 address of the machine in the **Network** field.
```

Comment:
Not sure if this is actually rendered on the site, but "application server's" has a smart instead of a straight apostrophe.